### PR TITLE
Adding psql command to chef-server-ctl

### PR DESCRIPTION
**Pull request by andy-dufour**
This is a single new command for chef-server-ctl, allowing you to drop into a psql session using the various backend services (erchef, bifrost, ocid, pushy, and reporting).

### CODE REVIEW

**Comment by marcparadise** [comment]
I've been following this through its iterations, this looks good to me. Once you fix up the description TODO (maybe something like 'launches an interactve psql session with the service database you name'?) ,  :+1: to merge.

**Comment by andy-dufour** [comment]
Should be good now. I'll leave it open so you can take one last look before merge.

File: omnibus/files/private-chef-ctl-commands/psql.rb [hunk 1]
```diff
@@ -0,0 +1,63 @@
+# Copyright (c) 2014 Chef Software, Inc.
+# All Rights Reserved
```
Comment (stevendanna) [inline omnibus/files/private-chef-ctl-commands/psql.rb]: This should be out standard Apache 2 notice.

File: omnibus/files/private-chef-ctl-commands/psql.rb [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+  elsif running_config.nil?
+    STDERR.puts "[ERROR] Cannot connect to database if you haven't completed a reconfigure"
+    exit 1
+  end
+
+  db_config = running_config
+
+  if File.exists?("/etc/opscode-push-jobs-server/opscode-push-jobs-server-running.json")
+    db_config.merge!(JSON.parse(File.read("/etc/opscode-push-jobs-server/opscode-push-jobs-server-running.json")))
+      #require 'pry'; binding.pry
```
Comment (stevendanna) [inline omnibus/files/private-chef-ctl-commands/psql.rb]: Mind cleaning up this commented debug code?

**Comment by stevendanna** [comment]
Left some minor comments but otherwise excited to get this in, I love having this available in dvm, will be nice to have it everywhere.

**Pull request merged**